src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -29,11 +29,13 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
     version: { type: GraphQLString },
     maxWorksPerArtist: { type: GraphQLInt },
     marketable: { type: GraphQLBoolean },
+    includeAuctionArtworks: {
```

Comment:
**suggestion / non-blocking**: The current New Works for You recommendation currently include some artworks at auction. So it seems strange to me that the default value of `includeAuctionArtworks=false` would not filter those out.

What do you think of naming this arg to something like `onlyAtAuction` to trigger the auction-specific recs vs the existing mixed New Works for You recs?

## Example

![Screenshot 2024-03-20 at 17 15 12](https://github.com/artsy/metaphysics/assets/123595/d80425b1-9290-4695-bc03-8e8629538c75)
![Screenshot 2024-03-20 at 17 15 21](https://github.com/artsy/metaphysics/assets/123595/27fd741d-b0d7-4629-8860-457b1bf1bc5b)

Reply:
I was mostly following the existing language used for backfill, which is `includeBackfill`. But yes, I see how the `includeAuctionArtworks` isn't quite accurate!